rmd_files/RNAseq_report.Rmd

```diff
@@ -2155,10 +2153,21 @@ if ( dataset == "PAAD" || dataset == "PAAD-IPMN" || dataset == "PAAD-NET" || dat
   comp_cancer_group <- ext_cancer_group
 }
 
+##### Read in Dragen's mapping metrics file (if exists) to extract number of reads
+dragen_mapping_metrics <- paste(params$dragen_rnaseq, list.files(params$dragen_rnaseq, pattern="\\.mapping_metrics.csv$"), sep = "/")
```

Comment:
```suggestion
dragen_mapping_metrics <- list.files(params$dragen_rnaseq, pattern="\\.mapping_metrics.csv$", full.names = TRUE)
```

Reply:
Ah you'd also need to check that `params$dragen_rnaseq` isn't NULL. Which brings me to: do we want to remove all the bcbio-related stuff during the r-pkg refactor, and focus on Dragen exclusively?

Reply:
updated in https://github.com/umccr/RNAsum/pull/65/commits/1101f0ce2110d162e99fc7e352400e87a7463e23

> do we want to remove all the bcbio-related stuff during the r-pkg refactor, and focus on Dragen exclusively?

yes and yes. That's something on the back of my mind. I'd rather like to get rid of bcbio/dragen/umccrise prefixes and just focus on wts/wgs as the input files are fairly generic (counts/fusions/SVs/etc). You're right, may be as we are going through this exercise, we should start tackling this as well.

Reply:
Cool, been working on exactly that, will open a new PR probably next couple days.